google-cloud-storage/src/main/java/com/google/cloud/storage/Storage.java

```diff
@@ -2672,6 +2674,24 @@ Blob createFrom(
    */
   ReadChannel reader(BlobId blob, BlobSourceOption... options);
 
+  /**
+   * Downloads the given blob to the given path using specified blob read options.
+   *
+   * @param blob
+   * @param path
+   * @param options
+   */
+  void downloadTo(BlobId blob, Path path, Blob.BlobSourceOption... options);
```

Comment:
```suggestion
  void downloadTo(BlobId blob, Path path, BlobSourceOption... options);
```

The storage interface already has a BlobSourceOption type used by the other methods (`reader`, `readAllBytes`) we'll want to use the same thing here.